### PR TITLE
language picker implementation

### PR DESCRIPTION
**Pull request by sarayp**

### CODE REVIEW

File: saray/language-picker/icons/Language Picker.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                   English
+               </p>
+           </div>
+
+       </div>
+    </div>
+
+
+
+</body></html>
```
Comment (reiniergs) [inline saray/language-picker/icons/Language Picker.html]: what is this file?

File: saray/language-picker/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Language Picker</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="styles.css" />
+</head>
+<body>
+    <div class="container">
+       <div class="lang_picker">
```
Comment (reiniergs) [inline saray/language-picker/index.html]: this should an html list for accessibility reasons

File: saray/language-picker/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+           <div class="lang_picker_selected">
+               <img class="lang_picker_us_flag" src="icons/united-states.svg" alt="US flag">
+               <p class="lang_picker_language">
+                   English (US)
+               </p>
+           <div>
+               <img class="lang_picker_checkmark" src="icons/ic-check-24-px-blue.svg" alt="Selected icon">
+           </div>
```
Comment (reiniergs) [inline saray/language-picker/index.html]: the checkmark should be part of the list item

**Review by reiniergs — DISMISSED** [review]

**Review by TahimiLeonBravo — APPROVED** [review]

**Review by TahimiLeonBravo — APPROVED** [review]

File: saray/language-picker/styles.css [hunk 4]
```diff
@@ -24,49 +24,46 @@
...
+.lang-us-flag {
     height: 24px;
     margin: 0 16px 8px 0;
     width: 24px;
 }
 
-.lang_picker_language {
-    color: #2196f3;
+.lang-label {
+    color: rgba(0, 0, 0, 0.87);
```
Comment (TahimiLeonBravo) [inline saray/language-picker/styles.css]: put the fallback color to the rgba

**Review by TahimiLeonBravo — APPROVED** [review]

**Review by TahimiLeonBravo — APPROVED** [review]

**Pull request merged**